modules/twinklexfd.js

```diff
@@ -1398,7 +1401,7 @@ Twinkle.xfd.callback.evaluate = function(e) {
 	}
 
 	var query, wikipedia_page, wikipedia_api, logpage, params;
-	var date = new Date();
+	var date = new Date(); // XXX: avoid use of client clock
```

Comment:
```suggestion
	// XXX: avoid use of client clock, still used by TfD, FfF, and CfD
	var date = new Date();
```

Just for extra clarity.

Reply:
👍